templates/script/script.js

```diff
@@ -1,252 +1,219 @@
-    if(window.innerWidth < 768)
-    {
-        document.getElementById('form').style.width="100%";
-        document.getElementById('table_div').style.width="100%";
-        document.getElementById('name').style.width="100%";
-        document.getElementById('phone').style.width="100%";
-        document.getElementById('email').style.width="100%";
-    }
+/*jshint esversion: 6 */
 
+let elEmail = document.getElementById("email");
+let elName = document.getElementById("name");
+let elPhone = document.getElementById("phone");
+let btPost = document.getElementById('post');
+let btGet = document.getElementById('get');
+let btClear = document.getElementById('clear');
+let divTable = document.getElementById('table_div');
+let fForm = document.getElementById('form');
 
-  //adding button-event listiners
-    document.getElementById('name').addEventListener('blur', checkName);
-    document.getElementById('phone').addEventListener('blur', checkPhone);
-    document.getElementById('email').addEventListener('blur', checkEmail);
-    document.getElementById('post').addEventListener('click', postData);
-    document.getElementById('get').addEventListener('click', getData);
-    document.getElementById('clear').addEventListener('click', clearForms);
+SetListeners();
 
+//adding button-event listiners
+function SetListeners() {
+    "use strict";
+    elName.addEventListener('blur', checkName);
+    elPhone.addEventListener('blur', checkPhone);
+    elEmail.addEventListener('blur', checkEmail);
+    btPost.addEventListener('click', postData);
+    btGet.addEventListener('click', getData);
+    btClear.addEventListener('click', clearForms);
+}
 
+function checkName() {
+    "use strict";
```

Comment:
You can move "use strict" to the top of the file if you want strict mode in all functions.